roles/terminfo/tasks/main.yml

```diff
@@ -12,11 +12,25 @@
   changed_when: false
   check_mode: false
 
+- name: Read remote terminfo entry
+  ansible.builtin.command:
+    argv:
+      - "{{ terminfo_infocmp_cmd }}"
+      - -x
+      - "{{ terminfo_term_name }}"
+  register: terminfo_remote_infocmp_result
+  changed_when: false
+  failed_when: false
```

Comment:
I'm not keen on this. What if `infocmp` isn't found? Or some file permission error occurs?